apps/payments/next/app/api/dev/nestapp/restart/route.ts

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { reinitializeNestApp } from '@fxa/payments/ui/server';
+import { notFound } from 'next/navigation';
+import { NextResponse } from 'next/server';
+
+export const dynamic = 'force-dynamic';
+
+export async function GET() {
```

Comment:
This feels a little strange having a route to reinitialize the NestApp to get an instance of the NestApp with the latest code changes, but I couldn't come up with better approach. Open to ideas though!